Makefile

```diff
@@ -0,0 +1,19 @@
+build: migrate.up
+	go build main.go
+
+migrate.up:
+	cp .env.example .env
+	cd db
```

Comment:
remove this line, no need to change directory

Reply:
Got it.